src/main/java/core/basesyntax/Colours.java

```diff
@@ -0,0 +1,10 @@
+package core.basesyntax;
+
+public enum Colours {
```

Comment:
According to the Java naming conventions, the name of the enum itself should be singular rather than plural. So 'Colours' should be renamed to 'Colour' to represent a single instance of the enumeration.